sdk/communication/Azure.Communication.Common/tests/CommunicationTestEnvironment.cs

```diff
@@ -25,5 +31,25 @@ public class CommunicationTestEnvironment : TestEnvironment
         public string LiveTestAccessKey => Core.ConnectionString.Parse(LiveTestConnectionString).GetRequired("accesskey");
 
         public string CommunicationTestPhoneNumber => GetVariable(AzurePhoneNumber);
+
+        public bool ShouldIgnorePhoneNumbersTests => !TestPackagesEnabled.Equals(TestPackagesEnabledDefaultValue)
+            && !TestPackagesEnabled.Contains(PhoneNumberTestPackagesEnabled);
+
+        public bool ShouldIgnoreSMSTests => !TestPackagesEnabled.Equals(TestPackagesEnabledDefaultValue)
+            && !TestPackagesEnabled.Contains(SmsTestPackagesEnabled);
+
+        public bool ShouldIgnoreChatTests => !TestPackagesEnabled.Equals(TestPackagesEnabledDefaultValue)
+            && !TestPackagesEnabled.Contains(ChatTestPackagesEnabled);
+
+        public bool ShouldIgnoreIdentityTests => !TestPackagesEnabled.Equals(TestPackagesEnabledDefaultValue)
+            && !TestPackagesEnabled.Contains(IdentityTestPackagesEnabled);
+
+        private string TestPackagesEnabled => getTestPackageEnabled();
+
+        private string getTestPackageEnabled()
```

Comment:
nit: in C#, we use PascalCase for method names

Reply:
noted